docs/application/dotnet/guides/tizen.circularUI/IndexPage.md

```diff
@@ -0,0 +1,61 @@
+---
+uid: Tizen.Wearable.CircularUI.doc.IndexPage
+summary: IndexPage control guide
+---
+# IndexPage
+
+The `IndexPage` allows you to know in advance the number of pages you have when you configure the application on multiple pages. It also shows the number of pages you are viewing.
+It is an extension of [Xamarin.Forms.MultiPage](https://developer.xamarin.com/api/type/Xamarin.Forms.MultiPage%3CT%3E/). When [Page](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/controls/pages) is added and removed in the [Xamarin.Forms.MultiPage](https://developer.xamarin.com/api/type/Xamarin.Forms.MultiPage%3CT%3E/), the number of dot marks increases or decreases automatically at the top of window.
+It is similar to [CarouselPage](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/controls/pages#carouselpage) in Xamarin.Forms, with the addition of `Index`.
+When [Page](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/controls/pages) is scrolled, `Index` operates select internally.
+
+![](data/IndexPage_action.png)
+
+## To add IndexPage
+
+To create a new index component, use the following XAML code.
```

Comment:
Please get this sentence above the code.

This Sentence is misleading

Reply:
I remove it.